liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/ServerFeatureSupport.java

```diff
@@ -191,6 +213,91 @@ private static boolean isSubModule(MavenProject potentialTopModule, MavenProject
         }
         return false;
     }
+    
```

Comment:
Shouldn't we delete the older checkMultiModuleConflicts that you took most of this logic from?  (Sorry, I feel like you raised this point but I don't see it in the PR.)   That'd give us an easier history to follow and less code is always easier to maintain.

Reply:
I kept it not knowing if it would be useful at any point in the future. I can remove and we can always find the logic in the git history if need be.